src/ensembl/src/header/Header.scss

```diff
@@ -61,25 +61,14 @@
   }
 }
 
-.aboutEnsembl {
+.topRight {
```

Comment:
`.topbarRight`? To complement the existing topbarLeft?

![image](https://user-images.githubusercontent.com/6834224/121398526-5de6be80-c94d-11eb-9448-e5cb46bfd2e7.png)